src/components/ArtisanSwitch/index.jsx

```diff
@@ -0,0 +1,66 @@
+import { useState } from 'react';
+import cls from 'classnames';
+import { Switch } from '@headlessui/react';
+import styles from './styles.module.css';
+
+import {
+  flex,
+  padding,
+  borders,
+  colors,
+  layout,
+} from '../../theme';
+
+export default function ArtisanSwitch({ switchLabel, setToggle }) {
+  const [enabled, setEnabled] = useState(false);
```

Comment:
So I think with a component like this, you almost always want it to be "controlled" via props. Which means, you should have setToggle and enabled both be props and use that to control things.

Reply:
Generally you don't want two different states which _mean_ the same thing because that will cause you all kinds of headaches trying to keep them in sync.